app/models/katello/concerns/widget_extensions.rb

```diff
@@ -0,0 +1,23 @@
+module Katello
+  module Concerns
+    module WidgetExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        SUBSCRIPTION_TEMPLATES = %w[subscription_status_widget subscription_widget].freeze
+
+        scope :without_subscription_widgets, -> { where.not(template: ::Widget::SUBSCRIPTION_TEMPLATES) }
+
+        singleton_class.send(:alias_method, :old_available, :available)
```

Comment:
A better way to do this is to use 'prepend':  https://www.justinweiss.com/articles/rails-5-module-number-prepend-and-the-end-of-alias-method-chain/

We have an example of the usage here:  https://github.com/Katello/katello/blob/master/lib/katello/engine.rb#L225   
where by it defines a method in the extension that overrides the original method, and you can call super() within it to call the original method.